src/api/features.api.js

```diff
@@ -60,7 +60,7 @@ export const identify = (layer, coordinate, mapExtent, screenWidth, screenHeight
     }
     axios
       .get(
-        `${API_BASE_URL}/rest/services/${layer.getTopicForIdentifyAndTooltipRequests()}/MapServer/identify`,
+        `${API_BASE_URL}rest/services/${layer.getTopicForIdentifyAndTooltipRequests()}/MapServer/identify`,
```

Comment:
just a general note (not related to this exact change): you might want to explicitly name the API you consider, so `CHSDI_BASE_URL` (or following the pattern you use below for what3words `CHSDI_API_BASE_URL`), this will make transition easier in the future when parts slowly move away from chsdi (as already happened e.g. with shortlink etc.)

Reply:
that's one way of looking at it, I was more thinking of "here is `https://api3.geo.admin.ch`, I do not care what is behind with routing, but that's my API base URL".
In the end, they will all be served under one roof, isn't it?
IMO, the least the front is aware of internal backend routing, the better

Reply:
in an ideal world yes ;-) I'm afraid we won't get completely away with this, but we can also adapt once we know more.